src/cli/mod.rs

```diff
@@ -43,6 +44,17 @@ enum Command {
     Repl(ReplArgs),
     /// Verifies a Lurk proof
     Verify(VerifyArgs),
+    /// Instansiates a new circom gadget to interface with bellperson.
+    ///
+    /// To setup a new circom gadget `<NAME>`, place your circom files in a designated folder and
+    /// create a file called `<NAME>.circom`. `<CIRCOM_FOLDER>/<NAME>.circom` is the input file
+    /// for the `circom` binary; in this file you must declare your circom main component.
+    ///
+    /// Then run `lurk circom --name <NAME> <CIRCOM_FOLDER>` to instansiate a new gadget `<NAME>`.
+    /// The new components are stored in `.lurk/circom/<NAME>/*`.
+    // (TODO: replace `.lurk` with env var)
+    #[command(verbatim_doc_comment)]
+    Circom(CircomArgs),
```

Comment:
Unfortunately a text this big will pollute the help menu

Reply:
`help` can show help for other subcommands, see https://github.com/clap-rs/clap/issues/416

Reply:
I understand. What I'm saying is that this wall of text will take the entire space for `lurk help`. We better have a short version and then a long version for `lurk circom --help`

Reply:
Will do